cmd/daemon/main.go

```diff
@@ -312,6 +311,7 @@ func main() {
 		}
 	}
 	daemonEvents.Subscribe(analytics)
+	daemonEvents.Settings.Publish(cfg)
```

Comment:
wouldn't this create generate events at every daemon restart, for the same user? or this is what is intended?